dockerfiles/cosmos-exporter/v0.3.0/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.16
+EXPOSE 9300
+
+RUN wget -q https://github.com/solarlabsteam/cosmos-exporter/releases/download/v0.3.0/cosmos-exporter_0.3.0_Linux_x86_64.tar.gz &&\
+    tar xvfz cosmos-exporter* -C /usr/bin
+
+ENTRYPOINT [ "cosmos-exporter" ]
+CMD [ "--denom", "uknow" ]
```

Comment:
```suggestion
```

I think we shouldn't taint it with OKP4 configurations.

Reply:
Yes effectively !
Moreover it will be overwritten by the kubernetes "args" anyway :+1: